doc/snippets/Microsoft.Data.SqlClient/SqlDataReader.xml

```diff
@@ -979,12 +979,12 @@ The <xref:Microsoft.Data.SqlClient.SqlDataReader.GetSchemaTable%2A> method retur
         An attempt was made to read or access columns in a closed <see cref="T:Microsoft.Data.SqlClient.SqlDataReader" />.
       </exception>
       <exception cref="T:System.InvalidCastException">
-        The retrieved data is not compatible with the <see cref="T:Microsoft.Data.SqlTypes.SqlVectorFloat32" /> type.
+        The retrieved data is not compatible with the <see cref="T:Microsoft.Data.SqlTypes.SqlVector" /> type.
```

Comment:
This cref may not be valid - keep an eye during docs review to address issues here.